test/gui/shared/steps/steps.py

```diff
@@ -646,3 +672,24 @@ def step(context):
 def step(context):
     for tabName in context.table:
         test.vp(tabName[0])
+
+
+@Then('the error "|any|" should be displayed')
+def step(context, errorMessage):
+    sharingDialog = SharingDialog()
+    test.compare(sharingDialog.getErrorText(), errorMessage)
+
+
+@When("the user closes the sharing dialog")
+def step(context):
+    clickButton(waitForObject(names.sharingDialog_Close_QPushButton))
```

Comment:
This has not been used anywhere. So, you can remove this.

Reply:
nice find. Thanks